.env

```diff
@@ -1,18 +1,20 @@
-ELASTICSEARCH_VERSION=8.8.0
+ELASTICSEARCH_VERSION=8.12.2
 IMAGE_REPO=ghcr.io/zammad/zammad
 MEMCACHE_SERVERS=zammad-memcached:11211
-MEMCACHE_VERSION=1.6.20-alpine
+MEMCACHE_VERSION=1.6.25-alpine
 POSTGRES_DB=zammad_production
 POSTGRES_PASS=zammad
 POSTGRES_USER=zammad
 POSTGRES_HOST=zammad-postgresql
 POSTGRES_PORT=5432
-POSTGRES_VERSION=15.3-alpine
+POSTGRES_VERSION=15.6-alpine
+POSTGRESQL_OPTIONS=?pool=50
 REDIS_URL=redis://zammad-redis:6379
-REDIS_VERSION=7.0.5-alpine
+REDIS_VERSION=7.2.4-alpine
 RESTART=always
 # Use a fixed version. You are responsible to update this to newer patch level versions yourself.
-VERSION=6.2.0-1
+# VERSION=6.3.0-0
+VERSION=develop
```

Comment:
TODO: change to static version after release.

Reply:
No, this MR is only a preperation before the release, which is scheduled in a few weeks. I will merge it only after, and switch the version to the commented out one `6.3.0-0`.

I created this MR so that we can test the changes before, and are ready when the release comes. Sorry for making that not clear enough.

Reply:
Ah, ok. Thanks for clarification :)
So beside app version it looks good to me as CI runs successfull.
I'll approve as soon as the correct version is set :)